src/olympia/addons/tasks.py

```diff
@@ -96,11 +106,7 @@ def unindex_addons(ids, **kw):
     for addon_id in ids:
         log.info('Removing addon [%s] from search index.' % addon_id)
         try:
-            es.delete(
-                AddonIndexer.get_index_alias(),
-                AddonIndexer.get_doctype_name(),
-                addon_id,
-            )
+            es.delete(index=AddonIndexer.get_index_alias(), id=addon_id)
         except TransportError:
             # We ignore already deleted add-ons.
             pass
```

Comment:
I can't find tests covering this bit (*), and it seems important - not only we want to guarantee that `unindex_addons()` works and that the document(s) are gone from the ES index, but also we want to make sure it works for already deleted/disabled add-ons that are modified.

Reply:
https://github.com/mozilla/addons-server/pull/19506/commits/dadfd88cd86b95a19d7708387a7f7fec21501281 adds a test to check `unindex_objects` works and doesn't fail.  It's a bit basic - it just counts records - but it works and doesn't raise.